src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Cardstrings.json

```diff
@@ -1,37 +1,38 @@
 {
   "${ModID}:CrescentSweep": {
     "NAME": "Crescent Sweep",
-    "DESCRIPTION": "Gain !B! Block. NL ${ModID}:Manifest ${ModID}:Crescent_Moon."
+    "DESCRIPTION": "Gain !B! Block. NL ${ModID}:Manifest ${ModID}:Crescent_Moon.",
+    "UPGRADE_DESCRIPTION": "Gain !B! Block. NL ${ModID}:Manifest ${ModID}:Crescent_Moon. NL ${ModID}:Inscribe a random card in your hand."
   },
   "${ModID}:MysticFlourish": {
     "NAME": "Mystic Flourish",
-    "DESCRIPTION": "Deal !D! damage. NL Activate the passive effect of your next orb.",
-    "UPGRADE_DESCRIPTION": "Deal !D! damage. NL Activate the passive effect of your next orb twice."
+    "DESCRIPTION": "Deal !D! damage.",
+    "UPGRADE_DESCRIPTION": "Deal !D! damage. NL ${ModID}:Inscribe a random card in your hand."
   },
   "${ModID}:CrystalMoonlight": {
     "NAME": "Crystal Moonlight",
     "DESCRIPTION": "${ModID}:Manifest a random orb. NL Exhaust",
-    "UPGRADE_DESCRIPTION": "${ModID}:Manifest !M! random orbs. NL Exhaust"
+    "UPGRADE_DESCRIPTION": "${ModID}:Manifest !M! random orbs."
   },
   "${ModID}:FullMoonHalo": {
     "NAME": "Full Moon Halo",
-    "DESCRIPTION": "Gain !B! Block. NL Draw a card. NL Manifest ${ModID}:Full_Moon."
+    "DESCRIPTION": " Gain !B! Block. NL Manifest ${ModID}:Full_Moon."
   },
   "${ModID}:MoonlightBarrage": {
     "NAME": "Moonlight Barrage",
-    "DESCRIPTION": "Activate the passive effect of all your orbs. NL Deal !D! damage to a random enemy for each."
+    "DESCRIPTION": "Deal !D! damage to a random enemy for each."
```

Comment:
what?

Reply:
need to word it like Barrage. forgot the Moonlight Barrage relied on having it's effect spelled out. and doesn't work with Inscribed out the box